src/dev/GraphiQL.tsx

```diff
@@ -67,25 +67,42 @@ const httpFetcher: Fetcher = (graphQLParams) =>
 const httpMultipartFetcher: Fetcher = (graphQLParams) =>
   ({
     subscribe: (...args: SubscribeArguments) => {
+      const abortController = new AbortController();
       const sink = getSinkFromArgs(args);
       const isIntrospectionQuery = args.length === 3;
 
-      fetchMultipart("http://localhost:4000/graphql", {
-        method: "POST",
-        body: JSON.stringify(graphQLParams),
-        headers: {
-          "content-type": "application/json",
-        },
-        onNext: (parts) => {
-          // Introspection is broken if we return a array instead of a single item.
-          // TODO: This should be addressed inside GraphiQL
-          sink.next(isIntrospectionQuery ? parts[0] : parts);
-        },
-        onError: sink.error,
-        onComplete: sink.complete,
-      });
+      try {
+        (async () => {
+          const response = await fetch("http://localhost:4000/graphql", {
+            method: "POST",
+            body: JSON.stringify(graphQLParams),
+            headers: {
+              "accept": "application/json, multipart/mixed",
+              "content-type": "application/json",
+            },
+            signal: abortController.signal,
+          });
+
+          const patches = await meros(response);
+          // @ts-ignore we need to ignore coz typescript don't understand this.
+          if (patches[Symbol.asyncIterator] < 'u') {
+            for await (const patch of patches) {
```

Comment:
nice api

Reply:
Can we please replace `if (patches[Symbol.asyncIterator] < 'u') {` with a check that is clear without reading up on it 😅. For demonstration purposes, I wanna keep it as easy to understand as possible :)